recipes/markdown2/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "markdown2" %}
+{% set version = "2.3.1" %}
+{% set sha256 = "4463dd754e79e4afd490c21d81db67b85b488a0d0adf2fee2b99a69c4e67d80e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+
+test:
+  imports:
+    - markdown2
+
+  commands:
+    - markdown2 --help
+
+about:
+  home: https://github.com/trentm/python-markdown2
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: A fast and complete Python implementation of Markdown
+
+extra:
+  recipe-maintainers:
+    - jakirkham
+    - pelson
```

Comment:
Went ahead and merged. If you decide you don't want to help maintain this, we can sort it out at the feedstock.